geometry/point.hpp

```diff
@@ -81,6 +81,9 @@ class Point final {
   friend class geometry::BarycentreCalculator;
 };
 
+template<typename Vector>
+using Segment = std::pair<Point<Vector>, Point<Vector>>;
```

Comment:
I'm somewhat reluctant to put that in point.hpp; for now let's keep it in perspective which is the only place where we use it, and see how things evolve from there